docs/application/tizen-studio/native-tools/manifest-text-editor.md

```diff
@@ -672,6 +688,265 @@ This element contains the settings for the widget class. After adding this eleme
+
+**Attributes:**
+
+- `appid`
+
+  Application unique ID (string)
+
+  You can use the a\~z, A\~Z, 0\~9, ".", "-", and "\_" characters, and the value must be less than 50 characters.
+
+  This can be used for launching or terminating the application explicitly.
+
+- `exec`
+
+  Application executable file path (string)
+
+- `type`
+
+  Tizen application type (available values: `capp`, `c++app`, `webapp`, `dotnet`)
+  > **Note**
+  >
+  > This attribute is read-only. Do not attempt to modify it.
+
+**For example:**
+
+```xml
+<component-based-application appid="org.tizen.component-based" exec="component-based" type="capp">
+   <label>component-based</label>
+   <icon>icon.png</icon>
+   <app-control>
+     <operation name="http://tizen.org/appcontrol/operation/share"/>
+   </app-control>
+   <metadata key="http://tizen.org/metadata/testkey" value="testvalue"/>
+   <datacontrol access="ReadOnly" providerid="http://uiapp.com/datacontrol/provider/component-based" type="Sql"/>
+   <splash-screens>
+     <splash-screen src="icon.jpg" type="img" dpi="hdpi" orientation="portrait" indicator-display="true"/>
+   </splash-screens>
+   <frame-component id="org.tizen.frame-component" main="true" icon-display="true" taskmanage="true" launch_mode="single">
+     <label>frame-component</label>
+     <icon>frame.png</icon>
+   </frame-component>
+   <service-component id="org.tizen.service-component" main="false">
+     <label>service-component</label>
+     <icon>service.png</icon>
+   </service-component>
+</component-based-application>
+```
+
+#### \<label\> Element
+
+This element contains the component-based application text.
+
+**Occurrences:**
+
+- 1 or more
+
+**Expected value:**
+
+- Label value in string
+
+**For example:**
+
+```xml
+<label>component-based</label>
+```
+
+#### \<icon\> Element
+
+This element contains the component-based application icon image.
+
+**Occurrences:**
+
+- 1 or more
+
+**Expected value:**
+
+- Icon file name
+
+**For example:**
+
+```xml
+<icon>icon.png</icon>
+```
+
+#### \<frame-component\> Element
+
+This element contains the settings for the frame component. After adding this element, you can use  `component_based_app_add_frame_component()` to create frame component instances in the component-based application.
+
+**Occurrences:**
+
+- 1 or more (optional)
+
+**Expected children:**
+
+| Child element    | Occurrences |
+|------------------|-------------|
+| `<label>`        | 1 or more   |
+| `<icon>`         | 1 or more   |
+
+**Attributes:**
+
+- `id`
+
+  Frame component unique ID (string)
+
+- `main`
+
+  Indicates which component is the main component (available values: `true`, `false`)
+
+- `launch_mode`
+
+  Component launch mode (available values: `single` (launched as a main component), `group` (launched as a sub component), `caller` (caller component [defines the launch mode](../../native/guides/app-management/app-controls.md#mode) with `app_control_set_launch_mode()`))
```

Comment:
Please remove one ")" at the end.

(caller component [defines the launch mode](../../native/guides/app-management/app-controls.md#mode) with `app_control_set_launch_mode()`)

Reply:
Done